scenarios/kubernetes_e2e.py

```diff
@@ -391,6 +391,7 @@ def main(args):
         runner_args.append('--test')
 
     cluster = cluster_name(args.cluster, os.getenv('BUILD_NUMBER', 0))
+    runner_args.append('--cluster=%s' % cluster)
```

Comment:
you can remove the `CLUSTER_NAME=` below, probably we don't need `E2E_NAME` either.